src/backend/blockchain/tests/functional/test_routes.py

```diff
@@ -26,44 +26,60 @@ def test_client():
 
     ctx.pop()
 
+@pytest.fixture(scope='module')
+def initial_chain():
+    return [src.Block(0, src.Transaction("", "", 0, 0, "", ""), 0,
+                              "0000", "0", "miner_id", 0)]
+
+
+@pytest.fixture(scope='module')
+def test_block_payload():
+    payload = {"from": "user1", "to": "user2", "amount": 20, 
+                "timestamp": 404, "id": "this_is_the_id", "signature": "fake_signature_lol", 
+                "minerId": "user3", "proof": "already_validated", "nonce": 10}
+    return payload
+
+@pytest.fixture(scope='module')
+def test_transaction_payload():
+    return {'from': "fake_wallet_id",
+            'amount': 0,
+            'to': "another",
+            'timestamp': 0, "id": "abc", "signature": "123"}
+        
```

Comment:
This is one of the major refactors, taking the stubs out of method and putting it in fixtures for reuse